pyproject.toml

```diff
@@ -2,13 +2,15 @@
 name = "rpaframework-meta"
 version = "0.0.1"
 description = "Metapackage for RPA Framework"
-authors = [
-	"RPA Framework <rpafw@robocorp.com>",
-]
+authors = ["RPA Framework <rpafw@robocorp.com>"]
 
 [tool.poetry.dependencies]
 python = "^3.7"
-rpaframework = { path = "packages/main", extras = ["cv", "playwright", "aws"] }
+rpaframework = { path = "packages/main", extras = [
+	"cv",
+	"playwright",
+	"aws",
+], develop = true }
```

Comment:
Rebuilt my meta venv with this branch and it works like it should!
```
(.venv) cmin@robocorp-mac rpaframework % python -c "import RPA.Robocorp.WorkItems as w; print(w)"
<module 'RPA.Robocorp.WorkItems' from '/Users/cmin/Repos/rpaframework/packages/main/src/RPA/Robocorp/WorkItems.py'>
```

Just to keep in mind that only `rpaframework` gets installed for development in this meta venv, the rest of the deps (including our `extras`) are installed normally.